scripts/get_workspace_packs.py

```diff
@@ -0,0 +1,14 @@
+import glob
+import json
+import os
+
+def get_workspace_packs(root):
+    # Find the packs by globbing using the 'provide' patterns in the manifest.
+    os.chdir(root)
+    with open('.codeqlmanifest.json') as manifest_file:
```

Comment:
If we choose to replace `.codeqlmanifest.json` with a `codeql-workspace.yml` we need to update this reference.